index.html

```diff
@@ -0,0 +1,113 @@
+    <title>Youtube Clone</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel=“stylesheet”>
+</head>
+<body>
+    <div class="navbar">
+
+        <div class="nav-logo">
+            <i class="material-icons">menu</i>
+            <img id="logo" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/Logo_of_YouTube_%282015-2017%29.svg/1280px-Logo_of_YouTube_%282015-2017%29.svg.png">
+        </div>
+
+        <div class="search-container">
+            <input class="search-input" type="text" placeholder="Search">
+            <button class="search-button" type="button"><i class="material-icons">search</i></button>
+        </div>
+
+        <div class="navbar-items">
+            <a><i class='material-icons'>apps</i></a>
+            <a><i class='material-icons'>message</i></a>
+            <a><i class='material-icons'>video_call</i></a>
+            <div class="navbar-avatar"></div>
+        </div>
+    </div>
+    <div class="main">
+        <div class="section-video-avatar">
+            <div class="video">
+                <iframe width="1024" height="640" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                <h1>Contributing to Open Source Part I: The Easy Way</h1>
+                <div class="menu-below-video">
+                    <span class="text-views">500,000 views</span>
+                    <div class="menu-container">
+                            <button class="video-action" type="button"><i class="material-icons" >thumb_up</i></button>
+                            <button class="video-action" type="button"><i class="material-icons">thumb_down</i></button>
+                            <button class="video-action" type="button"><i class="material-icons">share</i></button>
+                    </div>
+                </div>
+                <div class="video-info">
+                    <div class="video-info">
+                        <img src="https://yt3.ggpht.com/a/AGF-l79PHE32UB5d2fnmwsiJXu3vv_fnC_SNO899VQ=s88-c-k-c0xffffffff-no-rj-mo" alt="avatar">
+                        <div class="info">
+                            <span class="user-name">Pedro</span>
+                            <br>
+                            <p class="video-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto omnis doloremque itaque, illo voluptas quo vitae blanditiis nesciunt cumque libero?</p>
+                        </div>
+                    </div>
+                    <div>
+                        <button class="subscribe-button" type="button">SUBSCRIBE</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="section-sidebar">
```

Comment:
[OPTIONAL]
It's also best practice to use the `aside` tag for the side section. All these 3 changes are related to best practices and semantic HTML(very important topic)